test/WebJobs.Script.Tests/Rpc/LanguageWorkerChannelTests.cs

```diff
@@ -108,6 +109,14 @@ public void SendWorkerInitRequest_PublishesOutboundEvent()
             Assert.True(traces.Any(m => string.Equals(m.FormattedMessage, _expectedLogMsg)));
         }
 
+        [Fact]
+        public void SendSystemLogMessage_PublishesSystemLogMessage()
```

Comment:
- add test for isRpcSystemLog
- add tests for different categories - warning and error as well
- add test where Rpc LogLevel does not convert to a valid LogLevle